articles/container-apps/networking-configuration.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Networking configuration in Azure Container Apps environment
+description: Learn how to configure networking in Azure Container Apps.
+services: container-apps
+author: craigshoemaker
+ms.service: azure-container-apps
+ms.topic:  conceptual
+ms.date: 10/03/2024
+ms.author: cshoe
+---
+
+# Networking configuration in Azure Container Apps environment
+
+Azure Container Apps run in the context of an environment, with its own virtual network (VNet). This VNet creates a secure boundary around your Azure Container Apps [environment](environment.md). This article tells you how to configure your VNet.
+
+## HTTP edge proxy behavior
+
+Azure Container Apps uses the [Envoy proxy](https://www.envoyproxy.io/) as an edge HTTP proxy. Transport Layer Security (TLS) is terminated on the edge and requests are routed based on their traffic splitting rules and routes traffic to the correct application.
+
+HTTP applications scale based on the number of HTTP requests and connections. Envoy routes internal traffic inside clusters.
+
+Downstream connections support HTTP1.1 and HTTP2 and Envoy automatically detects and upgrades connections if the client connection requires an upgrade.
+
+Upstream connections are defined by setting the `transport` property on the [ingress](azure-resource-manager-api-spec.md#propertiesconfiguration) object.
+
+### Ingress configuration
+
+Under the [ingress](azure-resource-manager-api-spec.md#propertiesconfiguration) section, you can configure the following settings:
+
+- **Accessibility level**: You can set your container app as externally or internally accessible in the environment. An environment variable `CONTAINER_APP_ENV_DNS_SUFFIX` is used to automatically resolve the fully qualified domain name (FQDN) suffix for your environment. When communicating between container apps within the same environment, you may also use the app name. For more information on how to access your apps, see [Ingress in Azure Container Apps](./ingress-overview.md#domain-names).
```

Comment:
Where does the term "Accessibility level" come from? It's not in the API or portal. This is the setting you're talking about, right?
![image](https://github.com/user-attachments/assets/0cf9f486-b6a7-4ec1-bdf7-7f851d683202)

Reply:
The rest of the paragraph is unrelated to the ingress setting.

Reply:
Renamed "Accessibility level" to "ingress/ingress traffic" bullets to match the portal UI.
Moved the rest of that paragraph to the ingress-overview.md line 104.